poppy/fresnel.py

```diff
@@ -534,7 +534,7 @@ def propagate_direct(self, z):
 
```

Comment:
hmm, I can't seem to add line comments to lines outside of the highlighted diff region. Not sure if that's possible on here. I would like to comment on line 530: 
 `quadphase_2nd = np.exp(1.0j * k * z_direct) / (1.0j * self.wavelength * z_direct) * np.exp(`
Right now in my testing, `quadphase_2nd` ends up with units of 1/wavelength, because `self.wavelength` is a Quantity but `k, z_direct, x,` and `y` are all plain ndarrays implicitly in meters. So I think that needs to become `self.wavelength.to(u.m).value`